ironfish/src/fileStores/config.ts

```diff
@@ -473,8 +461,6 @@ export class Config extends KeyStore<ConfigOptions> {
       poolMaxConnectionsPerIp: 0,
       poolLarkWebhook: '',
       jsonLogs: false,
-      explorerBlocksUrl: 'https://explorer.ironfish.network/blocks/',
-      explorerTransactionsUrl: 'https://explorer.ironfish.network/transaction/',
```

Comment:
I feel okay with removing this from the SDK config, since it's just used as a UI feature. Though this makes me think that it'd be nice to have a way to have config settings that are CLI-specific, and also a way to have mapped configs...

Had you thought about including them in the network definition? Not really sure whether we'd want that since it again is more SDK-focused

Reply:
Yea, I've thought this before too having a separate CLI config. Another config value that should be CLI specific for example is `dataDir` or node `rpcTcpHost` and `rpcTcpPort`. Really those should have separate values for the CLI that tell the CLI which node to connect to. We just happen to re-use the node (server side) config values for connection which works most of the time but also results in some weird edge cases. So I definitely agree with CLI config but maybe we save for a separate PR.

Network definition is interesting yea but ultimately I agree I think its too CLI specific to add there